### PR TITLE
Update what-is-an-activity.md

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
Hotjar Feedback Response #374
## Notes to reviewers
Cleaned up a few pieces around terms and added headers and a note.

### CODE REVIEW

File: docs/concepts/what-is-an-activity.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
 Temporal does not recover Activity state in case of failures.
 Therefore an Activity function is allowed to contain any code without restrictions.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:27]: Therefore, an Activity Definition has no restrictions on its contents. (and link to the page!)

File: docs/concepts/what-is-an-activity.md [hunk 2]
```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
...
 
-Activities are invoked asynchronously through task queues.
-A task queue is essentially a queue used to store an Activity task until it is picked up by an available worker.
-The worker processes an Activity by invoking its implementation function.
+#### Invoke Activities through Task Queues
+
+Activities are invoked asynchronously through Task Queues.
+A Task Queue is essentially a queue used to store an Activity task until it is picked up by an available Worker.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:32]: I feel like this sentence can be merged with the one before it, and then you can discuss Workers.

File: docs/concepts/what-is-an-activity.md [hunk 3]
```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
...
+The Worker processes an Activity by invoking its implementation function.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:33]: So it runs an Activity Execution?

File: docs/concepts/what-is-an-activity.md [hunk 4]
```diff
@@ -39,12 +44,14 @@ It is possible to implement an Activity fully asynchronously by completing it fr
 
 Temporal does not impose any system limit on Activity duration. It is up to the application to choose the timeouts for its execution.
 
-Activities are dispatched to workers through task queues.
-Task queues are queues that workers listen on.
-Task queues are highly dynamic and lightweight.
-They don't need to be explicitly registered. And it is okay to have one task queue per worker process. It is normal to have more than one Activity type to be invoked through a single task queue. And it is normal in some cases (like host routing) to invoke the same Activity type on multiple task queues.
+Activities are dispatched to Workers through Task Queues.
+Task Queues are queues that Workers listen on.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:48]: We explained this point about Activities, Workers, and Task Queues before this, so please remove.

File: docs/concepts/what-is-an-activity.md [hunk 5]
```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
...
+Activities are invoked asynchronously through Task Queues.
+A Task Queue is essentially a queue used to store an Activity task until it is picked up by an available Worker.
+The Worker processes an Activity by invoking its implementation function.
 When the function returns, the worker reports the result back to the Temporal service which in turn notifies the Workflow about completion.
+
+#### Implement an Activity asynchronously
+
 It is possible to implement an Activity fully asynchronously by completing it from a different process.
 
 - An Activity can be implemented as a synchronous method or fully asynchronously involving multiple processes.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:40]: This sentence isn't necessary. The user coming here most likely knows about sync vs async

File: docs/concepts/what-is-an-activity.md [hunk 6]
```diff
@@ -39,12 +44,14 @@ It is possible to implement an Activity fully asynchronously by completing it fr
...
+Task Queues are queues that Workers listen on.
+Task Queues are highly dynamic and lightweight.
+They don't need to be explicitly registered. And it is okay to have one Task Queue per Worker process. It is normal to have more than one Activity type to be invoked through a single task queue. And it is normal in some cases (like host routing) to invoke the same Activity type on multiple Task Queues.
 
-Here are some use cases for employing multiple Activity task queues in a single Workflow:
+#### Employ Activity Task Queues in a single Workflow
+
+Here are some use cases for employing multiple Activity Task Queues in a single Workflow:
 
 - _Flow control_. A worker that consumes from a task queue asks for an Activity task only when it has available capacity. So workers are never overloaded by request spikes. If Activity executions are requested faster than workers can process them, they are backlogged in the task queue.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:56]: Don't forget to capitalize Temporal terminology

File: docs/concepts/what-is-an-activity.md [hunk 7]
```diff
@@ -63,7 +70,11 @@ Long running Activities can be used as a special case of leader election. Tempor
...
 
-note Cancellations are not immediate
+:::note
+
+Cancellations are not immediate.
+
+:::
 
 `ctx.Done()` is only signaled when a heartbeat is sent to the service.
 Temporal's SDK throttles this so a heartbeat may not be sent to the service until 80% of the heartbeat timeout has elapsed.
```
Comment (DJSanti) [inline docs/concepts/what-is-an-activity.md:80]: A Heartbeat may not be sent until 80% of the timeout has elapsed.

**Review by flossypurse — CHANGES_REQUESTED** [review]
It's awesome that you addressed the user feedback.

However, this whole page needs to be re-written from the ground up.

**Pull request closed without merging**